src/hooks/useGetData.jsx

```diff
@@ -0,0 +1,52 @@
+import { useState, useEffect, useCallback } from "react";
+import axios from "axios";
+import { renderHook } from '@testing-library/react-hooks'
+
+export const useGetData = () => {
+  const [date, setDate] = useState(new Date());
+  const [pictureData, setPictureData] = useState();
+  const [loading, setLoading] = useState(true);
+  const [errors, setErrors] = useState(false);
+
+  const fetchData = useCallback(async () => {
+    const dateString = () => {
+        if(date){
+          const currentDayOfMonth = date.getDate();
+          const currentMonth = date.getMonth();
+          const currentYear = date.getFullYear();
+          return currentYear + "-" + (currentMonth + 1) + "-" + currentDayOfMonth;
+        }
+        return '';
+    }

```

Comment:
this function could be defined as a util in a different file, when the `date` changes it is redefined but moving it to its own file and passing the `date` as param makes more sense